Dockerfile

```diff
@@ -6,6 +6,6 @@ RUN yum -y update && \
       yum -y install wget
 # Matching appsody binary does not exist in upstream.
 # Provide the proper version once it is available.
-RUN wget https://github.com/appsody/appsody/releases/download/0.X.0/appsody-0.X.0-1.x86_64.rpm
-RUN yum localinstall -y ./appsody-0.X.0-1.x86_64.rpm
+RUN wget https://github.com/appsody/appsody/releases/download/CLI_VERSION/appsody-CLI_VERSION-1.x86_64.rpm
+RUN yum localinstall -y ./appsody-CLI_VERSION-1.x86_64.rpm
```

Comment:
It would be better for this to be defined as an `ARG` line and passed to `docker build --build-arg`.

Reply:
I made that change and re-tested - we should be good to go.